src/components/VideoDetailsDescription/VideoDetailsDescription.component.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { VideoDescription } from './VideoDetailsDescription.styled';
+
+const formatDate = (date) => {
+  const d = new Date(date);
+  const ye = new Intl.DateTimeFormat('en', { year: 'numeric' }).format(d);
+  const mo = new Intl.DateTimeFormat('en', { month: 'short' }).format(d);
+  const da = new Intl.DateTimeFormat('en', { day: '2-digit' }).format(d);
+  return `${da}-${mo}-${ye}`;
+};
```

Comment:
you can have an `utils` folder if you need to reuse this code, sometimes it is also easier to test